plugins/author-name/client/components/Tooltip.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import cn from 'classnames';
+import styles from './Tooltip.css';
+import {t} from 'plugin-api/beta/client/services';
+import {Icon} from 'plugin-api/beta/client/components/ui';
+import moment from 'moment';
+
+export default ({className = '', memberSinceDate}) => (
+  <div className={cn(styles.tooltip, className)}>
+    <Icon name="date_range" className={styles.icon} />
+    <span className={styles.memberSince}>
+      {t('talk-plugin-author-name.member_since')}: {moment(memberSinceDate).format('MMMM DD, YYYY')}
```

Comment:
![image](https://user-images.githubusercontent.com/14221600/29609330-2317b2f0-87f7-11e7-85b7-26d4cf9d80ca.png)
According to `moment` documentation you must provide a format string as a second value: `moment(memberSinceData, dateFormat).format('MMMM DD, YYYY');`